Rover/sailboat.cpp

```diff
@@ -260,6 +264,52 @@ void Sailboat::get_throttle_and_mainsail_out(float desired_speed, float &throttl
         wingsail_out *= -1.0f;
     }
 
+    //
+    // direct mast rotation control
+    // configure SERVOx_MAX/MIN such that the mast rotates +-90 degrees
```

Comment:
i would like to see this hardcoded 90 a param, you could reuse the max sail angle.

Reply:
Changed to +- sail_angle_max and amended the parameter description

Reply:
the comment it now out of date ;)

Reply:
Thank you for noticing this. Taken out.